### PR TITLE
make: validate C format

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Signed-off-by: Akihiro Suda <suda.akihiro@lab.ntt.co.jp>

Options for `indent` are taken from https://github.com/opencontainers/runc/pull/950#discussion_r71268646

@cyphar PTAL?

### CODE REVIEW

File: libcontainer/nsenter/nsexec.c [hunk 1]
```diff
@@ -315,11 +314,11 @@ static int child_func(void *arg)
 	longjmp(*ca->env, ca->jmpval);
 }
 
-static int clone_parent(jmp_buf *env, int jmpval) __attribute__ ((noinline));
-static int clone_parent(jmp_buf *env, int jmpval)
+static int clone_parent(jmp_buf * env, int jmpval) __attribute__ ((noinline));
+static int clone_parent(jmp_buf * env, int jmpval)
```
Comment (cyphar) [inline libcontainer/nsenter/nsexec.c]: This change doesn't appear to make sense -- `jmp_buf *env` is the Linux kernel style, no? I'll have to double check why `indent` is doing that...

Comment (AkihiroSuda) [reply libcontainer/nsenter/nsexec.c]: According to man:

> You must use the ‘-T’ option to tell indent the name of all the typenames in your program that are defined by typedef.

Updated PR

**Comment by cyphar** [comment]
Should we be running the check as part of CI? Maybe you can add it to `.travis.yml`?

**Comment by AkihiroSuda** [comment]
It is already added. (travis -> make ci -> make validate -> validate-c)

**Comment by cyphar** [comment]
Ah, my apologies. Still LGTM.

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**